app/src/androidTest/java/ch/epfl/polybazaar/UI/SaleDetailsTest.java

```diff
@@ -337,5 +342,20 @@ public void testSetupViewMP() throws ExecutionException, InterruptedException {
         intended(hasComponent(MapsActivity.class.getName()));
         Intents.release();
     }
+
+    @Test
+    public void testClickOnViewPager() throws Throwable {
```

Comment:
Shouldn't there be an assertion in that test ? Even if it is assertThrows(onView(...)) ?